.travis.mk

```diff
@@ -87,8 +87,8 @@ deps_debian:
 		build-essential cmake coreutils sed \
 		libreadline-dev libncurses5-dev libyaml-dev libssl-dev \
 		libcurl4-openssl-dev libunwind-dev libicu-dev \
-		python python-pip python-setuptools python-dev \
-		python-msgpack python-yaml python-argparse python-six python-gevent \
+		python3 python3-pip python3-setuptools python3-dev \
+		python3-msgpack python3-yaml python-argparse python3-six python3-gevent \
```

Comment:
(See the comment for this goal a bit above in the code.)

I see that actual docker image contains Python 2 (+six/gevent/) and Python 3 (+modules). I would want to track actual state of the image here (so, don't remove Python 2 packages).

So, we should add `python3`, `python3-six`, `python3-gevent`, `python3-yaml`. We should NOT add `python3-pip`, `python3-setuptools`, `python3-dev`, `python3-msgpack`, because they're not used anywhere.

Reply:
It was about `docker.io/tarantool/testing:debian-stretch`. But everything is the same for `docker.io/tarantool/testing:debian-buster` (just checked).

Reply:
package list updated